azure/README.md

```diff
@@ -1,17 +1,17 @@
 # Prez Azure Function-App deployment files
 
-This directory contains the files required to deploy Prez as an Azure Function-App, as well as a Dockerfile that 
+This directory contains the files required to build and start or publish Prez as an Azure Function-App, as well as a Dockerfile that 
 can be used to build a container image for deploying the app as an Azure Container App.
 
 ## Publishing
-There is a publish.sh script that can be used to publish the app to Azure.
+There is a publish_or_start.sh script that can be used to either build and run the function app locally, or publish the app to Azure.
 To call it, make sure you are not in the "azure" directory, instead run the script from the root of the project.
 
 ```bash
-./azure/publish.sh --extra-options <function-app-name>
+./azure/publish_or_start.sh start|publish --extra-options <function-app-name>
```

Comment:
Change \<function-app-name\> to come after start|publish, but _before_ --extra options

Reply:
Updated in [2857b44](https://github.com/RDFLib/prez/pull/261/commits/2857b446056dcb816ebb462938db1ac0964afd5a)